api/v1/controllers/TeamController.js

```diff
@@ -2,28 +2,31 @@ var errors = require('../errors');
 var services = require('../services');
 var roles = require('../utils/roles');
 
+var bodyParser = require('body-parser');
+var middleware = require('../middleware');
 var router = require('express').Router();
 
 function joinTeam (req, res, next) {
   var email = req.auth.email;
-  var teamName = req.body['teamName'];
+  var teamName = req.body.teamName;
+
   return services.UserService.findUserByEmail(email)
   .then(function(user){
-    return services.TeamService
-      .addUserToTeam(user, teamName);
+	return services.TeamService
+	  .addUserToTeam(user, teamName);
   })
   .then(function(team){
-    return team.toJSON();
+	return team.toJSON();
```

Comment:
I see. It might be wise to have that in your docstring (I thought it just returned an `Object` instead of `Promise<Object>`